pyproject.toml

```diff
@@ -162,6 +157,15 @@ build = [
     "setuptools==70.0.0",
     "build==1.2.1"
 ]
+scripts = [
+    "jschema_to_python==1.2.3",
+    "psutil==5.9.2",
+    "stix2==3.0.1",
+    "sarif_om==1.0.4",
+    "requests==2.31.0",
+    "types-psutil==5.8.23",
```

Comment:
the `types` packages are to satisfy mypy, so i think they should go into the `dev` section.